arbitrum-docs/how-arbitrum-works/bold/gentle-introduction.md

```diff
@@ -137,7 +144,15 @@ If you’re intrigued by what BoLD can unlock for Arbitrum chains, we encourage
 
 ## Wen mainnet?
 
-BoLD is in `alpha`, which means there are a lot of planned improvements on the roadmap. A few high-level next steps for BoLD's journey to being deployed to Arbitrum chains include:
+:::caution Withdrawals leading up to a BOLD upgrade
+
+The confirmation timing on any withdrawal that is in-flight when the BOLD upgrade is activated will be delayed until the first BOLD assertion is confirmed. This means that for any Arbitrum chain that upgrades to use BOLD, including Arbitrum One and Arbitrum Nova, all pending withdrawals to L1 Ethereum that were initiated _before_ the upgrade will be delayed by 1 challenge period, plus the time between the withdrawal was initiated and the time that the BOLD upgrade takes place. This is because the upgrade effectively "resets" the challenge period for that are not yet finalized. 
```

Comment:
```suggestion
The confirmation timing on any withdrawal that is in flight when the BoLD upgrade is activated will be delayed until the first BoLD assertion is confirmed. This means that for any Arbitrum chain that upgrades to use BoLD, including Arbitrum One and Arbitrum Nova, all pending withdrawals to L1 Ethereum that were initiated _before_ the upgrade will be delayed by one challenge period, plus the time between the withdrawal was initiated and the time that the BoLD upgrade takes place. This is because the upgrade effectively "resets" the challenge periods that have not yet been finalized.
```